charts/sn-platform/values.yaml

```diff
@@ -945,6 +950,9 @@ broker:
       #  key: password
       #  name: kop-keystore-password
   # choose istio selector
+  aop:
+    enabled: false
+    proxyEnabled: false
```

Comment:
Would users prefer the proxy to be enabled by default?

Reply:
I think value's default is for most user who might not need aop enabled, so keep it to false by default.